test_settings.py

```diff
@@ -3,7 +3,8 @@
     'INSTALLED_APPS': [
         'djangocms_versioning',
         'djangocms_versioning.test_utils.polls',
-        'djangocms_versioning.test_utils.blogpost'
+        'djangocms_versioning.test_utils.blogpost',
+        'djangocms_versioning.test_utils.non_configured',
```

Comment:
I would just call this app people and possibly not put it in INSTALLED_APPS at all? I think the relationships app I created does create the models even without putting it in INSTALLED_APPS.